psi4/driver/procrouting/dft/superfunctionals.py

```diff
@@ -221,7 +221,7 @@ def test_ccl_functional(functional, ccl_functional):
         gamma_bb[index] = point['gamma_bb']
         index = index + 1
 
-    super = build_superfunctional(functional, N, 1)
+    super = build_superfunctional(functional, True, N, 1)
```

Comment:
Can you explicitly pass kwargs here? `(functional, True, npoints=N, deriv=1)`

Reply:
@obrien951, please edit when you get a chance.